src/Api/Endpoints/ImportNotifications/UpdatedImportNotificationRequest.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Defra.PhaImportNotifications.Api.Endpoints.ImportNotifications;
 
-internal sealed class UpdatedImportNotificationRequest
+public sealed class UpdatedImportNotificationRequest : IDateTimeRangeDefinition
```

Comment:
Should we mandate UTC from and to? As we'll be filtering downstream on UTC fields